src/components/FlyoutMenu/FlyoutMenu.js

```diff
@@ -0,0 +1,154 @@
+import { focusableElementDomString } from '@pdftron/webviewer-react-toolkit';
+import actions from 'actions';
+import classNames from 'classnames';
+import { findFocusableIndex } from 'core/accessibility';
+import getOverlayPositionBasedOn from 'helpers/getOverlayPositionBasedOn';
+import useMedia from 'hooks/useMedia';
+import useOnClickOutside from 'hooks/useOnClickOutside';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Swipeable } from 'react-swipeable';
+import selectors from 'selectors';
+import './FlyoutMenu.scss';
+
+const MENUS = [
+  'menuOverlay',
+  'groupOverlay',
+  'viewControlsOverlay',
+  'searchOverlay',
+  'signatureOverlay',
+  'zoomOverlay',
+  'redactionOverlay',
+  'toolStylePopup',
+];
```

Comment:
I just grabbed everything from all 3 of the converted menus, someone might have to check this over.

Reply:
Some of these might not exist any more. With this centralized FlyoutMenu component at least it's all in one place so we don't have to maintain 3 individual locations.